samples/extauthz/README.md

```diff
@@ -30,7 +30,7 @@ Note that `a` is just a default value for testing. It can be changed with the fl
     Send a check request with header `x-ext-authz: allow` to the Ext Authz server:
 
     ```console
-    $ kubectl exec -it $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -- curl -v ext-authz:8000 -H "x-ext-authz: allow"
+    $ kubectl exec -it $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -n foo -c sleep -- curl -v ext-authz:8000 -H "x-ext-authz: allow"
```

Comment:
At line 27, I do not see sleep pod being deployed in foo ns

Reply:
In https://istio.io/latest/docs/tasks/security/authorization/authz-custom/, it should use foo.

Reply:
That's why it show foo here.
![image](https://github.com/istio/istio/assets/17720035/60531289-ab45-44e5-a446-6583206033cb)

Two way to fix it.
1. add foo namespace everywhere.
2. remove foo namespace everywhere.

To make consistent with other doc, can I suggest to go to way 1. Thanks.